Doc/library/pathlib.rst

```diff
@@ -1592,6 +1592,23 @@ Copying, renaming and deleting
    .. versionadded:: 3.14
 
 
+.. method:: Path.move(target)
+
+   Recursively move this file or directory tree to the given *target*, and
+   return a new :class:`!Path` instance pointing to *target*.
+
+   If the *target* doesn't exist it will be created. If both this path and the
+   *target* are existing files, then the target is overwritten. If both paths
+   point to the same file or directory, or the *target* is a non-empty
```

Comment:
I'm not sure about the last point. You say that `.move` works as `shutil.move` but for the latter:

> Recursively move a file or directory (src) to another location and return the destination.
>  If dst is an existing directory or a symlink to a directory, then **src is moved inside that directory**. The destination path in that directory must not already exist.

So... is `move` not allowed to move a file like `mv file dir` and get `dir/file` ?